docs/cookbook/html-conversion.md

```diff
@@ -0,0 +1,980 @@
+            </div>
+        </div>
+    </article>
+{% endblock %}
+```
+:::
+<template v-slot:caption>
+modules/blog-page/views/show.html
+</template>
+</AposCodeBlock>
+
+As outlined above, the header of the blog piece pages is different from the other pages. There are two ways that we could approach this. We could create a dedicated blog header fragment and add it into our `views/fragments/header.html` file with a new name. The other approach is to add some conditional markup to our existing header fragment. In this case, we will do the latter.
+
+<AposCodeBlock>
+
+```django
+{% fragment headerArea(data) %}
+{% set background = apos.image.first(data.headerImage) %}
+
+{% set type = data.type %}
+
+<!-- Page Header-->
+{% if background %}
+{% set backgroundUrl = apos.attachment.url(background, { size: 'max' }) %}
+<header class="masthead" style="background-image: url('{{ backgroundUrl }}')">
+{% endif %}
+  <div class="container position-relative px-4 px-lg-5">
+    <div class="row gx-4 gx-lg-5 justify-content-center">
+      <div class="col-md-10 col-lg-8 col-xl-7">
+        <div class={{ "post-heading" if type === 'blog' else "site-heading"}}>
+          <h1>{{ data.heading }}</h1>
+          {% if type === 'blog' %}
+          <h2 class="subheading">{{ data.subheading }}</h2>
+          <span class="meta">Posted by <a href="{{ data.authorLink }}">{{ data.author }}</a> on {{ data.publicationDate }} </span>
+          {% else %}
+          <span class="subheading">{{ data.subheading }}</span>
+          {% endif %}
+        </div>
+      </div>
+    </div>
+  </div>
+</header>
+{% endfragment %}
+```
+
+<template v-slot:caption>
+views/fragments/header.html
+</template>
+</AposCodeBlock>
+
+So, what did we change? First, at the top, we created a new variable, `type`, and set it to the value of `data.type`. This is going to give us the type of page/piece that is being loaded. Looking at the code differences in the header section of the `post.html` template compared to the other three pages, the first thing we see is that the container for the header text has a class of `post-heading` instead of `site-heading`. In the code above, we conditionally set the class only if the page being loaded is of the type `blog`. Similarly, we only add the metadata if we are on this type of page.
+
+Now all that is left to do is add our pages to the site and give them some content. The "About Me" and "Contact Me" both use the `default-page` template. The existing "Home" page should be swapped out for a `blog-page` template. All that is left to do is create your blog articles!
+
+## Summary
+
+Any pre-made HTML template can be converted for use in Apostrophe through just two or three simple steps. First, add the styling to your Apostrophe project. Next, create a template for each of the template pages that substitutes data from schema fields into each area of the page that you want to edit. If needed, add special piece types and piece page types.
```

Comment:
```suggestion
Any pre-made HTML template can be converted for use in Apostrophe through a couple simple steps. First, add the front end assets to your Apostrophe project. Next, create a template for each of the template pages that substitutes data from schema fields into each area of the page that you want to edit. If needed, add special piece types and piece page types.
```

Maybe don't call out a number. There's 6 steps in this tutorial but yea it's pretty easy. We also added more than styling here. 

Should this also be a list instead?

```
Any pre-made HTML template can be converted for use in Apostrophe through a couple simple steps. 

- Add the front end assets to your Apostrophe project. 
- Create a template for each of the template pages that substitutes data from schema fields into each area of the page that you want to edit. 
- Add special piece types and piece page types.
```

Reply:
Changed - I liked the look of the bullet list. Thanks!